cristin-organization/src/main/java/no/unit/nva/cristin/organization/CristinOrgApiClientVersion2.java

```diff
@@ -0,0 +1,74 @@
+package no.unit.nva.cristin.organization;
+
+import static java.util.Arrays.asList;
+import static no.unit.nva.HttpClientProvider.defaultHttpClient;
+import static no.unit.nva.cristin.model.Constants.ORGANIZATION_PATH;
+import static no.unit.nva.cristin.model.Constants.UNITS_PATH;
+import static no.unit.nva.cristin.organization.QueryParamConverter.translateToCristinApi;
+import static no.unit.nva.model.Organization.ORGANIZATION_CONTEXT;
+import static no.unit.nva.utils.UriUtils.createCristinQueryUri;
+import static no.unit.nva.utils.UriUtils.getNvaApiUri;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import no.unit.nva.cristin.common.client.ApiClient;
+import no.unit.nva.cristin.common.client.IQueryApiClient;
+import no.unit.nva.cristin.model.SearchResponse;
+import no.unit.nva.cristin.organization.dto.version20230526.mapper.OrganizationFromUnitMapper;
+import no.unit.nva.cristin.organization.dto.version20230526.UnitDto;
+import no.unit.nva.model.Organization;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadGatewayException;
+
+public class CristinOrgApiClientVersion2 extends ApiClient
+    implements IQueryApiClient<Map<String, String>, Organization> {
+
+    public static final URI ORGANIZATION_ID_URI = getNvaApiUri(ORGANIZATION_PATH);
+
+    public CristinOrgApiClientVersion2() {
+        this(defaultHttpClient());
+    }
+
+    public CristinOrgApiClientVersion2(HttpClient client) {
+        super(client);
+    }
+
+    /**
+     * Fetch Organizations matching given query criteria using version 2 code.
```

Comment:
The version name is not v2, though

Reply:
What should it be called?

Reply:
And what should we call the packages containing the versions? Now it is "version20230526". Numeric only gives error